### ISSUE TITLE
Trying to scroll up repeatedly in chat history on Riot crashes the tab

### ISSUE THREAD

**Issue by bovine3dom** [tridactyl/tridactyl#1247]
Tridactyl 1.14.6-11-g0a07307, FF64.

Steps to reproduce:

- open riot.im/app
- click on the chat history
- hold k
- this kills the tab

**Comment by glacambre** [comment]
Can't reproduce. I get the "Tridactyl is slowing down Firefox" message but no crash :/

**Comment by bovine3dom** [comment]
Sorry, when I said crash, I meant freeze. It stops responding to input and must be killed with Ctrl-W.

**Comment by kxxvii** [comment]
Related: even if I just repeatedly tap some key to scroll I also get "Tridactyl is slowing down Firefox". Wouldn't say that I've crashed a tab or a browser session but I suspect that if I were to type drunk or something I would.

First thought it might be an issue with my key repeat settings — `xset r rate 450 70` — but that never happened before I installed Tridactyl and it doesn't happen in other applications. Not a huge problem — happens semi-regularly — but it's a nuisance.

Anyway, I just wanted confirm that I experience what @bovine3dom  and @glacambre is describing.
